data/campaigns/Sceptre_of_Fire/scenarios/Epilogue.cfg

```diff
@@ -227,11 +227,11 @@
         [/message]
         [message]
             speaker=Haldric II
-            message= _ "Your advice seems good to me. I will take it. As for you, and the gryphon, you are free to go. Alanin, you’ve served in the army long enough to retire, and even if you hadn’t, you’ve seen enough combat. So, go back to your village with your well-earned pension."
+            message= _ "Your advice is grating to my ears, not what I wish to hear...  And yet, I see the wisdom in it, and I agree.  Alanin, you’ve served in the army long enough to retire, and even if you hadn’t, you’ve seen enough combat. Go back to your village, and I shall see that you and your family live well and enjoy great success."
```

Comment:
There's a double-space in here, and there's also one in the "Aye, but it's no use" line.

Reply:
Sore subject, but I'll fix it, thanks.